spec/integration/outputs/templates_spec.rb

```diff
@@ -23,7 +23,9 @@
 
     elasticsearch_client.indices.delete_template(:name => '*')
     # This can fail if there are no indexes, ignore failure.
-    elasticsearch_client.indices.delete(:index => '*') rescue nil
+    elasticsearch_client.indices.delete(:index => '*') rescue puts("DELETE INDICES ERROR: #{$!}")
+    # Since we are pinned to ES client 7.x, we need to delete data streams the hard way...
+    elasticsearch_client.perform_request("DELETE", "/_data_stream/*") rescue puts("DELETE DATA STREAMS ERROR: #{$!}")
```

Comment:
In ES 8.x, deleting indices doesn't delete data streams, which was causing some of the specs here to pull in more results than they were intending if-and-only-if they were executed after specs that had created data streams without cleaning up after themselves.